packages/yourloops/components/switch-role/consent-dialog.tsx

```diff
@@ -98,7 +98,7 @@ function SwitchRoleConsentDialog(props: SwitchRoleConsentDialogProps): JSX.Eleme
           {t('button-decline')}
         </Button>
         <Button
-          id="switch-role-consent-dialog-button-accept"
+          data-testid='switch-role-consent-dialog-button-accept'
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏Double quotes should be used for strings in HTML code